integration/kubernetes/helm-chart/alluxio/templates/fuse/daemonset.yaml

```diff
@@ -109,7 +103,7 @@ spec:
                 command: ["/opt/alluxio/integration/fuse/bin/alluxio-fuse", "unmount", "/alluxio-fuse"]
           envFrom:
           - configMapRef:
-              name: {{ template "alluxio.name" . }}-config
+              name: {{ template "alluxio.fullname" . }}-config
```

Comment:
Changed to fullname or this will fail to match with the config. Same applies to all other resources.

Reply:
full name should not have the `release-name-` prefix